### ISSUE TITLE
File restores itself after deleting

### ISSUE THREAD

**Issue by m0byn** [saber-notes/saber#1068]
### Describe the bug

I am trying to delete a note. Unfortunately, whenever I sync after deleting it, it just restores itself. Tried it with several different devices and also clearing the cache - nothing helped. 

Could anyone help me out how to finally delete the note?

### To reproduce

1. Delete note
2. Sync
3. See that note restored itself

### Expected behavior

After deletion file should actually be deleted.

### Saber version

v0.18.5 (FOSS)

### Device

- Device: Samsung Galaxy Tab S6
- OS: Android 13 (LineageOS 20)

**Comment by rrjooss** [comment]
This also happens when folders are renamed.

To reproduce:
1. Rename folder
2. Sync
3. See the folder duplicated with its old name (not duplicated yk what I mean)

**Comment by dreamAviator** [comment]
as far as i know, the problem is kinda, that the app doesn‘t really delete stuff
i had a big nite, around 90 mb. i deleted it, and it didnt show up in the app, but it wasnt really deleted. it took up space in my nextcloud and on my device. i had to delete it on every debice manually to get it to be fully deleted

**Comment by m0byn** [comment]
So, it is not sufficient to delete it on the nextcloud server? One really has to delete it from every device? 

Did you sync between deleting from the devices?

**Comment by dreamAviator** [comment]
Yes. At least I had to delete it from every device, otherwise it restored itself.

Oct 11, 2025 11:51:07 m0byn ***@***.***>:

**Comment by m0byn** [comment]
Alright, I will give it a try.
